src/App.js

```diff
@@ -1,26 +1,9 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
 
-function App() {
+function App({ children }) {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <div>{children}</div>
```

Comment:
I'd recommend having the wrapping element on App view as a main tag. This way, you won't forget to have this on every view that is a child of App